Resources/ServerInfo/Guidebook/DeltaV/Medical/Pharmacist.xml

```diff
@@ -0,0 +1,40 @@
+<Document>
+# Pharmacist
+The "crème de la crème" in terms of knowledge in chemistry, the Pharmacist is the best at its domain! Whether they studied at Interdyne Pharmaceutics or the Andromeda Initiative Laboratories, their valuable knowledge is to be respected.
```

Comment:
Let us not insert unintelligible lore babble into the guidebook. Nobody knows what these companies are; they have not been documented. A pharmacist is also not the Super Chemist, and I do not want players to misunderstand our intentions here.